README.md

```diff
@@ -30,7 +30,12 @@
 
```

Comment:
- [ ] Kindly add the right screenshot in the 3rd query in the pull request summary because i can see that after fixing your time is less than 1000ms but i am not able to see what was it before because you haven't added the right screenshot there 😨 Kindly add the right screenshot 😊